.env

```diff
@@ -1,6 +1,7 @@
-REACT_APP_API_URL="http://192.168.109.145:8080"
+REACT_APP_API_URL="http://localhost:8080"
```

Comment:
There should be two or three ENV variables.
REACT_APP_DEVELOPMENT_API_URL="http://locahlost:8080"
REACT_APP_STAGING_API_URL="http://ec2-3-239-162-28.compute-1.amazonaws.com:8080"
REACT_APP_PRODUTION_API_URL="https://api.dialogbox.com/"


And in the code, read `NODE_ENV` and switch API_URL from it.